mods/d2k/weapons.yaml

```diff
@@ -593,6 +625,7 @@ Sound:
 			harvester: 50
 		DamageTypes: Prone50Percent, TriggerProne, SoundDeath
 	Warhead@3Eff: CreateEffect
+		Explosion: shockwave
```

Comment:
This makes an explosion appear on top of the tank a few hundred ms after it fires.  Was this intentional?